static/_app/tools/tradecraft/game/main.js

```
@@ -0,0 +1,480 @@
+	}
+	
+	document.ontouchmove = function(event){
+		event.preventDefault();
+	}
+	
+	
+	
+	
+	// audio
+	var names = ["good_trade", "bad_trade"];
+	for (var i = 0; i < names.length; ++i) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'i'. Consider to delete it or rename it.

'i' is defined here:


(lint/suspicious/noRedeclare)